float64[] latitudes
float64[] longitudes
string[] types
int32[] lefts
int32[] tops
int32[] ids
